src/pybind/mgr/pg_autoscaler/module.py

```diff
@@ -39,7 +40,8 @@ def nearest_power_of_two(n):
     # Low bound power of tow
     x = v >> 1
 
-    return x if (v - n) > (n - x) else v
+    return x if (v - n) > (n - x) and t == "Low" else v
```

Comment:
change def nearest_power_of_two(n, t='Low'): to def nearest_power_of_two(n, t='t'): 
and return x if (v - n) > (n - x) or t == "Low" else v